docs/Wetboek Tedeapolis.md

```diff
@@ -0,0 +1,562 @@
+# Wetboek Tedeapolis 2020
+
+## Wetstechnische informatie
+
+| | |
+|---|---|
+| Overheidsorganisatie      | Hoge Raad Tedeapolis |
+| Officiële naam            | Algemeen Wetboek Tedeapolis 2020 |
+| Citeertitel               | Wetboek Tedeapolis|
+| Vastgesteld door          | De Hoge Raad|
+| Onderwerp                 | Openbare orde en veiligheid|
+
+## Algemene Bepalingen
+
+### A1 - Reikwijdte
+
+1. Het Wetboek Tedeapolis (verder aangeduid als WTDA) is bedoeld als aanvulling op het Nederlands Recht en heeft betrekking op het eiland, de gemeente, en de stad Tedeapolis.
+2. Het Wetboek is leidend voor alle beschreven gevallen. Daarbuiten is het Nederlands Recht van toepassing.
+3. Het Wetboek is nauw verweven met de Algemene Plaatselijke Verordening Tedeapolis (APV). De APV biedt echter slechts een indicerend kader voor eventuele bestraffing in-game (“in RP”) en is verder bedoeld voor het afhandelen van straffen buiten de RP om.
+
+### A2 - Geldigheid
+
+1. Bij veranderingen van een wetgeving na het tijdstip waarop het feit begaan is, worden de voor de verdachte meest gunstigste bepalingen toegepast.
+2. Het is voor een staff-lid niet toegestaan om mee te beslissen over een zaak die over hunzelf, vrienden en/of familie gaat. Dit om te versterken dat een zaak op een eerlijke en rechtvaardige manier behandeld wordt.
+3. Een overtreding of misdrijf die niet opgenomen is in het Wetboek Tedeapolis, maar wel als zodanig bekend staat in het Nederland Recht, kan niet bestraft worden.
```

Comment:
wat wordt hier mee bedoeld?

Reply:
Voor 1. -> Als een wet wijzigt terwijl iemand daar op dat moment aan onderhevig is (tijdens verhor wordt er een commit gedaan en daarbij wordt een celstraf ineens lager), wordt de lagere straf toebedeeld.

Voor 2. -> Staff mag niet hun staff privilleges gebruiken voor RP doeleinden.

Voor 3. -> Staat het niet in het Wetboek Tedeapolis, dan is het (redelijkerwijs) niet strafbaar. Aan ons is het dan om het, indien nodig, zo snel mogelijk wèl op te nemen.